python/cudf/cudf/utils/ioutils.py

```diff
@@ -9,31 +9,42 @@
 import warnings
 from io import BufferedWriter, BytesIO, IOBase, TextIOWrapper
 from threading import Thread
-from typing import TYPE_CHECKING
+from typing import TYPE_CHECKING, Any
 
 import fsspec
 import fsspec.implementations.local
 import numpy as np
 import pandas as pd
+import pyarrow as pa
 from fsspec.core import expand_paths_if_needed, get_fs_token_paths
 
 import cudf
 from cudf.api.types import is_list_like
 from cudf.core._compat import PANDAS_LT_300
 from cudf.utils.docutils import docfmt_partial
+from cudf.utils.dtypes import np_dtypes_to_pandas_dtypes, np_to_pa_dtype
 
 try:
     import fsspec.parquet as fsspec_parquet
-
 except ImportError:
     fsspec_parquet = None
 
+try:
+    import ujson as json  # type: ignore[import-untyped]
+except ImportError:
+    import json
```

Comment:
*question*: Why this? I think we're only using this to load the (tiny) metadata json dict, so it seems perhaps unnecessary?

Reply:
I'm fine with just using `json`; I was just porting over the equivalent behavior